projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts

```diff
@@ -1,8 +1,10 @@
 import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
-import {expect, test} from '@playwright/test';
+import {expect, Locator, Page, test} from '@playwright/test';
 
 const {describe} = test;
 
+test.use({testIdAttribute: `automation-id`});
```

Comment:
I used this one to rename test attribute. Should I change name of test attributes right inside in the components or leave it like this?

Reply:
It is cool feature!
But we use Page Objects for our tests. They does not have direct access to `test.use`.

But I think you can add `testIdAttribute: "automation-id"` to this object (to enable it globally):
https://github.com/taiga-family/taiga-ui/blob/0a4bcf294896f0a5bc3064e593d74b545b12e126/projects/demo-playwright/playwright.config.ts#L25

And update all locators inside `/projects/demo-playwright/tests`:
<img width="1020" alt="Screenshot 2023-11-27 at 16 12 27" src="https://github.com/taiga-family/taiga-ui/assets/35179038/da2f8f1c-85b2-40cb-88de-316a12ad4655">

(but don't change `projects/demo-playwright/utils/page-objects` – they can published in future and should not depend on any global configs)